cmd/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	api "github.com/ChainSafe/gossamer/internal"
+	"os"
+
+	cfg "github.com/ChainSafe/gossamer/config"
+	"github.com/ChainSafe/gossamer/p2p"
+	"github.com/ChainSafe/gossamer/polkadb"
+	"github.com/ChainSafe/gossamer/rpc"
+	"github.com/ChainSafe/gossamer/rpc/json2"
+	log "github.com/inconshreveable/log15"
+	"github.com/naoina/toml"
+)
+
+func main() {
+	f, err := os.Open("./config.toml")
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		err = f.Close()
+		if err != nil {
+			log.Error("err closing conn", "err", err.Error())
+		}
+	}()
+
+	srvlog := log.New(log.Ctx{"blockchain": "gossamer"})
+	var config cfg.Config
+	if err = toml.NewDecoder(f).Decode(&config); err != nil {
+		srvlog.Warn("toml error::: %s", err.Error())
+	}
+	srv, err := p2p.NewService(config.ServiceConfig)
+	if err != nil {
+		srvlog.Warn("error starting p2p %s", err.Error())
+	}
+	srvlog.Info("🕸️ starting gossamer blockchain...", log.Ctx{"datadir": config.BadgerDB.Datadir})
+	srv.Start()
+
+	_, err = polkadb.NewBadgerDB(config.BadgerDB.Datadir)
```

Comment:
Please modify this to use the correct path (ie. remove the need for `../` as discussed below). Some considerations should be made for relative vs absolute paths. If you would like I can explore the specifics further but it seems that relative paths should be from the CWD (ie. the directory you are in when you run the `gossamer` command).